pyproject.toml

```diff
@@ -28,6 +28,7 @@ Homepage = "https://github.com/python-lsp/python-lsp-server"
 [project.optional-dependencies]
 all = [
     "autopep8>=1.6.0,<2.1.0",
+    "black>=23.7.0,<24.0",
```

Comment:
```suggestion
```

I don't think this is necessary because these are optional dependencies provided by the server for its users. And black formatting is not provided here but by the [`python-lsp-black`](https://github.com/python-lsp/python-lsp-black) plugin.

Reply:
Thanks for the insight!